linux/installation/mde_installer.sh

```diff
@@ -809,7 +809,7 @@ onboard_device()
     fi
 
     # validate onboarding
-    sleep 3
+    sleep 120
```

Comment:
Not a blocking comment but would be good to add a sleep with a shorter duration say 15 sec and keep checking for 10 times in a loop so we do exit early in case the health has turned true sooner.

Reply:
That is a much better suggestion than just having a flat sleep time. I have added a loop to break out if on-boarding occurs prior to the allocated period.